addons/mass_mailing/models/mass_mailing.py

```diff
@@ -235,7 +234,12 @@ def _get_total_mailings(self, cr, uid, ids, field_name, arg, context=None):
             required=True,
         ),
         'campaign_id': fields.many2one('utm.campaign', 'campaign_id', 
-            required=True, ondelete='cascade'),
+            required=True, ondelete='cascade',
+            help="This is a name that helps you keep track of your different campaign efforts. E.g.: Fall_Drive, Christmas_Special"),
```

Comment:
Actually I think e.g. has to be used directly in the sentence, without uppercase (I hate reviewers, e.g. Lilo).